mrp_unbuild_view_improves/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
```

Comment:
Wrong license comment

Reply:
Fixed, as AGPL

Reply:
Right one is LGPL-3. We'll only use AGPL-3 if an OCA addon with such license is involved

Reply:
Ok, noted and updated